buildpack/build_test.go

```diff
@@ -460,7 +478,7 @@ func testBuild(t *testing.T, when spec.G, it spec.S) {
 				})
 			})
 
-			it("should error when launch.toml is not writable", func() {
+			it.Pend("should error when launch.toml is not writable", func() { // TODO: ask Natalie - do we need this test?
```

Comment:
@natalieparellano

Reply:
I think so? This test was there before I did the refactor. Is there a reason to remove it?

Reply:
Yes, the way we fake that `launch.toml` is not writable is by making it a directory, and it makes us troubles with the new functionality.
@ekcasey and I were wondering whether we can delete this test or should we think about a different way to fake a not-writable file.
What do you think?

Reply:
I think we can just delete it.

Reply:
Thanks!
Please see https://github.com/buildpacks/lifecycle/pull/537/commits/9bd6a05415c17939ad6fa3105eae95823aac568f.